documentation/contributing/introduction.adoc

```diff
@@ -80,4 +82,57 @@ In contrast to the _Deploying_ guide, this guide provides a reduced set of instr
 
 The _Using Strimzi_ guide provides configuration instructions for Kafka components, and instructions for using Strimzi Operators.
 The majority of the content relates to how you might want to modify your deployment and introduce additional features, such as Cruise Control or distributed tracing.
-The _Using Strimzi_ guide also contains the API reference guide which is generated directly from the Java code of the operators.
+
+=== Custom Resource API Reference
+
+The _Using Strimzi_ guide (documentation/using/master.adoc) also contains a _Custom Resource API Reference_ that describes the configuration properties for custom resources.
+
+The _Custom Resource API Reference_ is built from two files.
+
+.Custom Resource API Reference files
+[source,asciidoc,options="nowrap"]
+----
+\include::modules/con-common-configuration-properties.adoc[leveloffset=+1]
+\include::modules/appendix_crds.adoc[]
+----
+
+The `con-common-configuration-properties.adoc` file contains descriptions of common configuration properties.
+The content for the `documentation/modules/appendix_crds.adoc` file is generated directly from descriptions in the Java code.
+
+Java files in the `api/` folder are annotated so that the descriptions are picked up in the build.
+
+.Java annotations for documentation
+[source,java,options="nowrap"]
+----
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.DescriptionFile;
+----
+
+The tables in `appendix_crds.adoc` are built from `@Description` annotations in the Java files.
+
+Additional information is included by adding:
+
+. An `@DescriptionFile` annotation to the Java file
+. A corresponding description file (`.adoc`) in the `documentation/api/` folder
+. An `include:__DESCRIPTION-FILE-NAME__` reference in `appendix_crds.adoc`
+
+For example, to add additional configuration for `KafkaUserQuotas`:
+
+. `api/src/main/java/io/strimzi/api/kafka/model/KafkaUserQuotas.java` contains:
+** `import io.strimzi.crdgenerator.annotations.Description`
+** `import io.strimzi.crdgenerator.annotations.DescriptionFile`
+** `@Description("_descriptions for individual properties..._");`
+** An `@DescriptionFile` annotation
+. `documentation/api` includes the `io.strimzi.api.kafka.model.KafkaUserQuotas.adoc` file containing the additional configuration description.
++
+The description file requires the same name as the related Java package.
+. `appendix_crds.adoc` contains a reference to include the additional configuration description:
++
+[source,asciidoc,options="nowrap"]
+----
+### `KafkaUserQuotas` schema reference
+
+/include::../api/io.strimzi.api.kafka.model.KafkaUserQuotas.adoc[leveloffset=+1]
+----
+
+If you change any anything in the `api` module of the Java code, you must rebuild the _Custom Resource API Reference_ using a xref:make-tooling[make command].
```

Comment:
Typo: any

You must rebuild the API Reference before submitting the PR?

Reply:
:+1:  I added to the build section

> If you change the Strimzi API, you need to regenerate the API Reference before submitting your PR by running the following from the root: